app/src/main/java/org/projectbuendia/client/ui/dialogs/EditPatientDialogFragment.java

```diff
@@ -98,16 +87,16 @@ private void populateFields(Bundle args) {
         LocalDate birthdate = Utils.toLocalDate(args.getString("birthdate"));
         if (birthdate != null) {
             Period age = new Period(birthdate, LocalDate.now());
-            mAgeYears.setText("" + age.getYears());
-            mAgeMonths.setText("" + age.getMonths());
+            mAgeYears.setText(String.valueOf(age.getYears()));
```

Comment:
Just a style difference then.  :)  Given two equivalent ways to express the same thing I'll usually pick the shorter one (less probability of causing a line break, which has a high cost).